src/hardware/vga_draw.cpp

```diff
@@ -2876,7 +2864,7 @@ void VGA_SetOverride(bool vga_override) {
 			vga.draw.vga_override=true;
 		} else {
 			vga.draw.vga_override=false;
-			vga.draw.width=0; // change it so the output window gets updated
+			vga.draw.render.width=0; // change it so the output window gets updated
```

Comment:
Style: missing spaces around `=`

Reply:
This is just a rename; not touching this file otherwise.